frontend_tests/casper_tests/06-settings.js

```diff
@@ -99,6 +90,15 @@ casper.then(function () {
     });
 });
 
+casper.then(function () {
+    casper.waitUntilVisible('#account-settings-status', function () {
+        casper.test.assertSelectorHasText('#account-settings-status', 'Updated settings!');
+
+        casper.click('[data-section="your-bots"]');
```

Comment:
Do you mean to ask if these casper tests failed in their original state? (when the get api box was moved from `your-bots` to `your-account` in the code but the tests weren't updated)? They were failing then.